.gitignore

```diff
@@ -63,5 +62,4 @@ cypress/screenshots
 
 # Local environment setup
 .env
-.vscode
```

Comment:
Why no .vscode in ignores?

Reply:
This line was duplicated (check line 18)

Reply:
doh! sorry!